lib_adat/src/adatReceiver-100.h

```diff
@@ -8,7 +8,7 @@
 #include <xs1.h>
 #include <stdio.h>
 #pragma unsafe arrays
-void adatReceiver48000(buffered in port:32 p, chanend oChan) {
+void adatReceiver48000(buffered in port:32 p, streaming chanend oChan) {
```

Comment:
In both lib_xua and sw_audio_analyser, the chanend passed to adatReceiver48000() is not a streaming chanend. 
https://github.com/xmos/lib_xua/blob/develop/lib_xua/src/core/main.xc#L306
https://github0.xmos.com/xmos-int/sw_audio_analyzer/blob/master/app_audio_analyzer_xcore200_mc/src/app_audio_analysis.xc#L264

If we change the API of adatReceiver48000(), will the calling functions need to change to pass a streaming chanend? I think we need to track it so we don't forget to change

Reply:
Ah yes, the checking will cause a build fail. Also the usage of outuint()/inuint() on a streaming chanend will also cause a build fail. We shall have to release this as a major version bump. Thanks for the heads up!

I think changing to type to its actual usage is ultimately the right thing to do, since it will allow for any proper checking in the future.

Reply:
Added  missing changelog item for this